Userland/Applications/FileManager/main.cpp

```diff
@@ -354,6 +394,11 @@ int run_in_desktop_mode([[maybe_unused]] RefPtr<Core::ConfigFile> config)
                 file_context_menu->add_action(directory_view.delete_action());
                 file_context_menu->add_separator();
 
+                if (node.full_path().ends_with(".zip")) {
```

Comment:
Let's ensure this also works with `ARCHIVE.ZIP`, i.e. let's do a case-insensitive compare.

```suggestion
                if (node.full_path().ends_with(".zip", CaseSensitivity::CaseInsensitive)) {
```